models/classes/LtiProvider/LtiProviderService.php

```diff
@@ -20,22 +20,27 @@
 
 namespace oat\taoLti\models\classes\LtiProvider;
 
+use oat\generis\model\OntologyAwareTrait;
 use oat\oatbox\service\ConfigurableService;
+use oat\taoDelivery\model\execution\DeliveryExecutionService;
+use oat\taoDeliveryRdf\model\ContainerRuntime;
+use oat\taoLti\models\classes\LtiException;
+use oat\taoLtiConsumer\model\delivery\form\NoLtiProviderException;
```

Comment:
Please avoid using any logic from `taoLtiConsumer` to prevent circular dependencies